pages/study/index.js

```diff
@@ -33,46 +42,70 @@ export default function StudyPage({ wordList, databaseMutate }) {
   const [isStudyMode, setIsStudyMode] = useState(false);
   const [isFront, setIsFront] = useState(true);
 
+  useEffect(() => {
+    function generateStudyMode() {
+      if (wordList) {
+        const entriesDue = wordList.filter((entry) => {
+          return isDue(entry.study.lastReview, entry.study.stage);
+        });
+
+        setStudyList(entriesDue);
+
+        return entriesDue;
+      }
+    }
+    generateStudyMode();
+  }, [wordList]);
```

Comment:
why do you need a useEffect here? Coudn't you just filter the wordList directly?

Reply:
I had it without a useEffect before. But it gave me an infinity loop.

Reply:
Ah but maybe you don't even need a State for your filtered list. just use the constant "entriesDue" directly

Reply:
It throws me an infinity loop again :(